xep-0321.xml

```diff
@@ -55,7 +55,7 @@
   <p>This document addresses the following requirements:</p>
   <ol>
     <li>Make it possible for remote services or entities to manage user's roster by the same mechanisms that descibed in the &rfc6121;.</li>
-    <li>Provide a way for user to control which services have permission to manage his roster.</li>
+    <li>Provide a way for user to control which services have permission to manage their roster.</li>
```

Comment:
I find it odd to continue the sentence "This document addresses the following requirements" with "provides foobar". I find "provide foobar" more natural.

Agreeing on the missing articles for "user". This, however, affects the whole XEP.